subt/ros/robot/src/filter_pointcloud.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python
+"""
+  Filter PointCloud2 received from X4 drone for Octomap server
+  - remove nearby propellers (up to 31cm) in the RGBD camera view
+  - replace xyz = (inf, inf, inf) to range larger than 10m
```

Comment:
Yes, it is fitted to X4 only. For me this is not "nice place to live", i.e. ROS and Python2, so it can end up with Octomap library wrapped into OSGAR node. But Freyja is the next robot to use similar filter, so parametrization will happen very soon (I would postpone it to next PR).